themes/courses.labster.com/lms/templates/ccx/enrollment.html

```diff
@@ -0,0 +1,154 @@
+        em_start=HTML('<em>'),
+        em_end=HTML('</em>'),
+        platform_name=settings.PLATFORM_NAME,
+    )}
+	<br /><br />
+	${_("Checking this box has no effect if 'Unenroll' is selected.")}
+      </p>
+    </div>
+  </div>
+
+  <div class="enroll-option">
+    <input type="checkbox" name="email-students" id="email-students" value="Notify-students-by-email" checked="yes" aria-describedby="email-students-helper">
+    <label style="display:inline" for="email-students">${_("Notify users by email")}</label>
+    <div class="hint email-students-hint">
+      <span class="hint-caret"></span>
+      <p class="text-helper" id="email-students-helper">
+	${Text(_("If this option is {em_start}checked{em_end}, users will receive an email notification.")).format(
+        em_start=HTML('<em>'),
+        em_end=HTML('</em>'),
+    )}
+      </p>
+    </div>
+  </div>
+
+  <div>
+    <input type="submit" name="enrollment-button" class="enrollment-button" value="${_("Enroll")}">
+    <input type="submit" name="enrollment-button" class="enrollment-button" value="${_("Unenroll")}">
+  </div>
+  <div class="request-response"></div>
+  <div class="request-response-error"></div>
+  </form>
+</div>
+
+<div class="member-lists-management" style="float:left;width:50%">
+  <form method="POST" action="ccx_manage_student" class="ccx-manage-student-form">
+  <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }">
+  <div class="auth-list-container active">
+    <div class="member-list-widget">
+      <div class="member-list">
+        <h2> ${_("Student List Management")}</h2>
+        %if messages:
+          <label for="ccx_std_list_messages" class="sr">${_("CCX student list management response message")}</label>
+          <div id="ccx_std_list_messages" tabindex="-1" class="request-response-error">
+            %for message in messages:
+              ${message}
+            %endfor
+          </div>
+        %endif
+        <table>
+          <thead>
+            <tr>
+              <td class="label">Username</td>
+              <td class="label">Email</td>
+              <td class="label">Revoke access</td>
+              <td class="label">Enrollment Status</td>
+            </tr>
+          </thead>
+          <tbody>
+            %for member in ccx_members:
+            <tr>
+              <td>${member.user}</td>
+              <td>${member.user.email}</td>
+              <td><a class="revoke"><span class="fa fa-times-circle" aria-hidden="true"></span> ${_("Revoke access")}</a></td>
+              <td>${_("Enrolled")}</td>
+            </tr>
+            %endfor
+            <!-- Start: Added by Labster -->
+            <!-- Add enrollment status -->
+            %for member in ccx_enrolls:
+            <% username = member.email.split('@')[0].lower() %>
+            <tr>
+              <td>${username}</td>
```

Comment:
Why is it handled differently?
`<% username = member.email.split('@')[0].lower() %>` in line 100
and `<td>${member.user}</td>` in line 91?

Reply:
because when teacher enroll a student that not registered on edx, we don't have any information about the username of student so I using part string of email as username.

Reply:
@lukyfebrianto it's better to use full `email` instead, because Instructors work with emails, but not with usernames. Part of email and real usernames may differ

Reply:
changed @polesye